src/translations/sqlb_de.ts

```diff
@@ -1694,7 +1694,7 @@ Sollen die bearbeiteten Daten auf Zeile=%1, Spalte=%2 angewendet werden?</transl
     <message>
         <location filename="../EditTableDialog.ui" line="595"/>
         <source>Check Constraints</source>
-        <translation>CHECK-Beschränkungen</translation>
+        <translation>Check-Beschränkungen</translation>
```

Comment:
This should probably stay consistent with line 1726.

Reply:
Ok, thank you for your reply.

Yes you are right. 
Cloud you please tell me how i change this (and your other comments) without to break more?

Sorry for the inconvenience!

Reply:
Simply run the `git pull` command on the branch you've been working on, edit it in the editor,
and then commit again to push it. Altenatively, you can use 'suggestion' feature like this:
```suggestion
        <translation>CHECK-Beschränkungen</translation>
```